narwhals/_spark_like/expr.py

```diff
@@ -201,6 +208,22 @@ def _alias(df: SparkLikeLazyFrame) -> list[Column]:
             kwargs={**self._kwargs, "name": name},
         )
 
+    def all(self) -> Self:
+        def _all(_input: Column) -> Column:
+            from pyspark.sql import functions as F  # noqa: N812
+
+            return F.bool_and(_input)
+
+        return self._from_call(_all, "all", returns_scalar=True)
```

Comment:
We simplified a bit the other methods, we can refactor as

```suggestion
        from pyspark.sql import functions as F  # noqa: N812

        return self._from_call(F.bool_and, "all", returns_scalar=True)
```